pkgs/applications/video/shotcut/default.nix

```diff
@@ -41,16 +40,14 @@ mkDerivation rec {
     mlt
     qtbase
     qtmultimedia
-    qtx11extras
     qtwebsockets
-    qtquickcontrols2
-    qtgraphicaleffects
+    fftw
   ];
 
   env.NIX_CFLAGS_COMPILE = "-I${mlt.dev}/include/mlt++ -I${mlt.dev}/include/mlt";
-  qmakeFlags = [
-    "QMAKE_LRELEASE=${lib.getDev qttools}/bin/lrelease"
-    "SHOTCUT_VERSION=${version}"
+  cmakeFlags = [
+    "CMAKE_LRELEASE=${lib.getDev qttools}/bin/lrelease"
+    "-DSHOTCUT_VERSION=${version}"
     "DEFINES+=SHOTCUT_NOUPGRADE"
   ];
```

Comment:
`qmakeFlags` can't just be turned into `cmakeFlags` by changing the attribute name, the two build managers don't have the same formats & ways of dong things. You'll need to adjust the values accordingly.

- `QMAKE_LRELEASE`: I think you should try dropping this outright instead of this rename, `qttools` in `nativeBuildInputs` should be enough for `lrelease` to be found & used?
- `SHOTCUT_VERSION`: Prepending `-D` as i suggested it to you is enough here, we want to define [a CMake variable](https://github.com/mltframework/shotcut/blob/5fb954d30506f68d7fd83b24779fe635986f1921/CMakeLists.txt#L7-L9) there.
- `DEFINES+=SHOTCUT_NOUPGRADE`: Trickier:
  This is to stop shotcut from running a new version check for users, as that's fairly useless in our case. But upstream doesn't provide any way to conveniently disable this, you're expected to pass through a C `#define`.
  In QMake, you could add the define by passing `DEFINES+=<name>=<val>`. With CMake the least-destructive way is likely to pass it as `-DSHOTCUT_NOUPGRADE` via the [`CXXFLAGS`](https://cmake.org/cmake/help/v3.27/envvar/CXXFLAGS.html) envvar before the first call to CMake (prolly in `preConfigure`?).
  Or, bypassing the need to mess with envvars, you could add `-DSHOTCUT_NOUPGRADE` to `env.NIX_CFLAGS_COMPILE` above to smuggle it into the compilation.
  <sub>I don't like any of these options, upstream should really just add a CMake `option` for this…</sub>